modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -18,7 +18,7 @@ You can route the traffic to multiple Ingresses through a single AWS Load Balanc
 +
 [source,yaml]
 ----
-apiVersion: elbv2.k8s.aws/v1beta1 <1>
+apiVersion: elbv2.k8s.aws/v1 <1>
```

Comment:
Only `AWSLoadBalancerController` API was promoted to `v1`, the other APIs remained in their verions.